pandas/core/frame.py

```diff
@@ -4749,7 +4749,7 @@ def drop(
         Remove rows or columns by specifying label names and corresponding
         axis, or by specifying directly index or column names. When using a
         multi-index, labels on different levels can be removed by specifying
-        the level.
+        the level. The now-unused levels remain in the MultiIndex._levels attribute.
```

Comment:
I think it would actually be better to just link to https://pandas.pydata.org/pandas-docs/stable/user_guide/advanced.html#defined-levels as you did in the original issue. I don't think it's great to advertise "private" attributes in the docs.

Reply:
Done